src/main/resources/assets/appliedenergistics2/lang/de_DE.lang

```diff
@@ -50,12 +51,36 @@ tile.appliedenergistics2.BlockCraftingAccelerator.name=Fertigungs-Prozessoreinhe
 tile.appliedenergistics2.BlockCraftingUnit.name=Fertigungseinheit
 tile.appliedenergistics2.BlockMolecularAssembler.name=Molekularassembler
 
+// Stairs
+tile.appliedenergistics2.ChiseledQuartzStairBlock.name=Gemeißelte Certusquartztreppe
+tile.appliedenergistics2.FluixStairBlock.name=Fluixtreppe
+tile.appliedenergistics2.QuartzPillarStairBlock.name=Certusquartzsäulentreppe
+tile.appliedenergistics2.QuartzStairBlock.name=Certusquartztreppe
+tile.appliedenergistics2.SkyStoneBlockStairBlock.name=Himmelssteinblocktreppe
+tile.appliedenergistics2.SkyStoneBrickStairBlock.name=Himmelssteinziegeltreppe
+tile.appliedenergistics2.SkyStoneSmallBrickStairBlock.name=Kleine Himmelssteinziegeltreppe
+tile.appliedenergistics2.SkyStoneStairBlock.name=Himmelssteintreppe
+ 
+// Slabs
+tile.appliedenergistics2.ChiseledQuartzSlabBlock.name=Gemeißelte Certusquartzstufe
+tile.appliedenergistics2.FluixSlabBlock.name=Fluixstufe
+tile.appliedenergistics2.QuartzPillarSlabBlock.name=Certusquartzsäulenstufe
+tile.appliedenergistics2.QuartzSlabBlock.name=Certusquartzstufe
+tile.appliedenergistics2.SkyStoneBlockSlabBlock.name=Himmelssteinblockstufe
+tile.appliedenergistics2.SkyStoneBrickSlabBlock.name=Himmelssteinziegelstufe
+tile.appliedenergistics2.SkyStoneSmallBrickSlabBlock.name=Kleine Himmelssteinziegelstufe
+tile.appliedenergistics2.SkyStoneSlabBlock.name=Himmelssteinstufe
+
+// Chat Messages
 chat.appliedenergistics2.ChestCannotReadStorageCell=ME-Truhe kann die Speicherzelle nicht einlesen.
 chat.appliedenergistics2.SettingCleared=Einstellung gelöscht.
 chat.appliedenergistics2.OutOfRange=Drahtloser Zugangspunkt außer Reichweite.
 chat.appliedenergistics2.InvalidMachine=Ungültige Maschine.
 chat.appliedenergistics2.LoadedSettings=Einstellungen erfolgreich geladen.
 chat.appliedenergistics2.DeviceNotPowered=Gerät hat zu wenig Energie
+chat.appliedenergistics2.DeviceNotWirelessTerminal=Gerät is keine Drahtloskonsole.
+chat.appliedenergistics2.DeviceNotLinked=Gerät is nicht verbunden.
```

Comment:
Oops, fixed!

Thanks for having a look! Is there a chance you could check my other open translation PRs?
OpenMods/OpenBlocks#733
jaquadro/StorageDrawers#181